activemq/tests/compose/docker-compose.yaml

```diff
@@ -0,0 +1,12 @@
+version: "3"
+
+services:
+  namenode:
+    image: rmohr/activemq:${ACTIVEMQ_VERSION}
+    container_name: dd-test-activemq-server
+    environment:
+      ACTIVEMQ_SUNJMX_START: "-Dcom.sun.management.jmxremote.port=1616 -Dcom.sun.management.jmxremote.ssl=false -Dcom.sun.management.jmxremote.authenticate=false"
+    ports:
+      - "61616:61616"
```

Comment:
Need this?

Reply:
We include it in `activemq_xml` integration, so kept it for consistency.  Also it features as part of the explicit docker commands in the image docs:
https://hub.docker.com/r/rmohr/activemq/